### PR TITLE
Support GHC-9.0.2

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ See also https://pvp.haskell.org/faq
 ## Version 1.4.0.2
 
  * Restore older GHC support
+ * Suppor GHC-9.0.2
```
Comment (andreasabel) [inline CHANGES.md:6]: Typo: "Suppor[t]"

**Review by andreasabel — APPROVED** [review]

File: CHANGES.md (hunk 1 above)
Comment (phadej) [reply CHANGES.md]: Thanks

**Comment by bergmark** [comment]
👍  looks like this is the main thing needed for stackage to use 9.0.2

**Comment by juhp** [comment]
Is it possible to revise hashable-1.3.5.0 to allow ghc-bignum-1.1, please?

**Comment by phadej** [comment]
As soon as there is GHC-9.0.2 in ghcup metadata.

**Comment by bergmark** [comment]
@phadej could you make the revision for 1.3.5.0 please? It would help us out a lot with the 9.0.2 upgrade in stackage.

**Comment by phadej** [comment]
@bergmark sure, https://hackage.haskell.org/package/hashable-1.3.5.0/revisions/ done.

**Comment by bergmark** [comment]
🙇 🙇 🙇

**Pull request merged**